controllers/services/create.go

```diff
@@ -35,6 +35,10 @@ func Create(au models.User, s models.ServiceInput, definition, body []byte, isAn
 		return 400, []byte("Environment name does not support char '" + models.EnvNameSeparator + "' as part of its name")
 	}
 
+	if st, res := h.IsAuthorizedToResource(&au, h.UpdateProject, dt.GetType(), s.Datacenter); st != 200 {
```

Comment:
To create a build, the user needs to have permissions to both update a project and an environment (checked bellow)?